src/python/RucioUtils.py

```diff
@@ -124,5 +124,5 @@ def getTapeRecallUsage(rucioClient=None, account=None):
     
     rules = rucioClient.list_replication_rules(filters=filters)
     usage = sum(getRuleQuota(rucioClient, rule['id']) for rule in rules\
-                if rule['state'] in ['REPLICATING', 'STUCK', 'SUSPENDED'])  # in Bytes
+                if rule['state'] in ['OK', 'REPLICATING', 'STUCK', 'SUSPENDED', 'INJECT', 'WAITING_APPROVAL'])  # in Bytes
```

Comment:
if you want to select all possible states, why not simply remove this line ?
In any case, although we use automatic approve here, rules in WAITING_APPROVAL should not be counted as "using space". I am not sure about rules in INJECTING, i.e. whether it comes before or after APPROVAL.

Reply:
I didn't remove the line as i still wanted to test that only one of these 6 states are being used and not any random value. Acc to Hasan, INJECT is the first step and it comes before WAITING_APPROVAL, followed by REPLICATING.

Reply:
So we should not count INJECT either. Or account separately for "pendind rules". But let's try not to make it too complicated